src/Control/Director.php

```diff
@@ -76,40 +80,47 @@ class Director implements TemplateGlobalProvider
     private static $alternate_base_folder;
```

Comment:
> unless we don't use this base_folder to construct a URL...

We don't. :)

```php
/**
 * Returns the root filesystem folder for the site. It will be automatically calculated unless
 * it is overridden with {@link setBaseFolder()}.
 *
 * @return string
 */
public static function baseFolder()
{
    $alternate = Director::config()->uninherited('alternate_base_folder');
    return ($alternate) ? $alternate : BASE_PATH;
}
```

Nothing to do with URLS.